corehq/apps/reports/static/reports/js/case_list.js

```diff
@@ -1,9 +1,26 @@
+hqImport("hqwebapp/js/initial_page_data").get('user_types')
```

Comment:
i don't think you want to import these at the top. try putting `"hqwebapp/js/initial_page_data"` in the list of imported modules after kissmetrix and adding it as a param.

Reply:
https://github.com/dimagi/commcare-hq/pull/22079/commits/428d727fed2307f40b6b6b129fdc3fec446711a2